### PR TITLE
add env-file

### PR DESCRIPTION
**Pull request by anagas**
Add Env-file as option

### CODE REVIEW

**Comment by anagas** [comment]
format code

**Comment by anagas** [comment]
Hello @oliver006 

Is the Coverage decreased (-0.5%) okey for you or should i add this 2 lines?

**Comment by oliver006** [comment]
Thanks for the PR!
Can you explain a little more what that new command line parameter does and what its used for? 
And yes, adding a test that confirms your code works as intended is a good idea.

File: main.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ type Function struct {
 
 	EnvironmentDelimiter string              `json:"environment_delimiter"`
 	Environment          []map[string]string `json:"environment"`
-
+	EnvironmentFile      string              `json:"env_vars_file"`
```
Comment (oliver006) [inline main.go:34]: Can you bring back the empty line plz? 
And a question about the `json` tag: this means the yaml will be `env_vars_file: ...` - right? 
Can you maybe name the struct field accordingly? Make it `EnvVarsFile` ? All others match the json tag.

**Comment by anagas** [comment]
Hello Oliver006

I want to add this feature:

--env-vars-file=FILE_PATH
Path to a local YAML file with definitions for all environment variables. All existing environment variables will be removed before the new environment variables are added.

**Comment by oliver006** [comment]
> I want to add this feature:
> 
> --env-vars-file=FILE_PATH
> Path to a local YAML file with definitions for all environment variables. All existing environment variables will be removed before the new environment variables are added.

Ok, great!

**Comment by anagas** [comment]
@oliver006 I also added the feature vpc-connector.

File: .env.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FOO: bar
```
Comment (oliver006) [inline .env.yaml:1]: Can you please remove this file or put it in a folder called `contrib` and link to it?

File: main.go [hunk 3]
```diff
@@ -28,10 +28,12 @@ type Function struct {
 	Source               string `json:"source"`
 	Timeout              string `json:"timeout"`
 	ServiceAccount       string `json:"serviceaccount"`
+	VpcConnector         string `json:"vpcconnector"`
 
 	EnvironmentDelimiter string              `json:"environment_delimiter"`
 	Environment          []map[string]string `json:"environment"`
 
+	EnvironmentVarsFile string `json:"envvarsfile"`
```
Comment (oliver006) [inline main.go:36]: `json:"env_vars_file"` ?

**Review by oliver006 — COMMENTED** [review]
Looks good! 🎉 
Last thing missing: can you add the two new parameters to the example in the README?

**Review by oliver006 — APPROVED** [review]
Looks good - thanks!

**Comment by oliver006** [comment]
Released as https://github.com/oliver006/drone-gcf/releases/tag/v1.11.0

**Comment by anagas** [comment]
Thank you!

**Pull request merged**